linode/manager
# M3-5259: Referral page changes

## Description

Changes to reflect the upcoming referral threshold:
- If a user has not spent $25, display message and hide referral link
- If a user has spent $25 but has not referred anyone yet, show referral link but hide results
- If a user has spent $25 and has referred people, show referral link and results

Created a local variable that currently represent if users are allowed to make referrals

**Remaining tasks:**
- [x] Replace local variable with actual variable from API
- [x] Remove code for testing purposes

## How to test

- Go to `/profile/referrals`
- Make sure each state matches the corresponding designs including when the flag is turned off
- Uncomment the corresponding code to see the different states if your test account doesn't have any referrals
